.travis.yml

```diff
@@ -4,8 +4,8 @@ language: php
 sudo: false
```

Comment:
Unrelated, but this can now be removed:
```suggestion
```
Re: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration